src/human_test.ts

```diff
@@ -0,0 +1,193 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import * as human from './human';
+
+const assert = chai.assert;
+
+describe('The human functions', () => {
+  function testPad() {
+    const testCases: [number, number, string][] = [
+      [0, 0, '0'],
+      [1, 1, '1'],
+      [10, 1, '10'],
+      [10, 2, '10'],
+      [10, 3, '010'],
+      [31558150, 8, '31558150'],
+      [31558150, 9, '031558150']
+    ];
+    for (const testCase of testCases) {
+      assert.equal(human.pad(testCase[0], testCase[1]), testCase[2]);
+    }
+  }
+
+  it('should return padded integers from pad', testPad);
+
+  function testStrDuration() {
+    const testCases: [number, string][] = [
+      [0, '  0s'],
+      [1, '  1s'],
+      [-1, '  1s'],
+      [2, '  2s'],
+      [10, ' 10s'],
+      [-30, ' 30s'],
+      [59, ' 59s'],
+      [60, '  1m'],
+      [-61, '  1m  1s'],
+      [123, '  2m  3s'],
+      [3599, ' 59m 59s'],
+      [3600, '  1h'],
+      [3601, '  1h  1s'],
+      [3659, '  1h 59s'],
+      [3660, '  1h  1m'],
+      [3661, '  1h  1m  1s'],
+      [86399, ' 23h 59m 59s'],
+      [86400, '  1d'],
+      [86401, '  1d  1s'],
+      [604799, '  6d 23h 59m 59s'],
+      [604800, '  1w'],
+      [31558150, ' 52w  1d  6h  9m 10s']
+    ];
+    for (const testCase of testCases) {
+      assert.equal(human.strDuration(testCase[0]), testCase[1]);
+    }
+  }
+
+  it('should return human-readable duration from strDuration', testStrDuration);
+
+  function testDiffDate() {
+    const now = 1584972056 * 1000; // 03/23/2020 @ 2:00pm (UTC)
+    const testCases: [number, string][] = [
+      [0, '0s'], //                 0s
```

Comment:
Nit: seems like the linter/formatter messed up the alignment in these comments.

Reply:
Done.